src/Transports.AspNetCore/WebSockets/SubscriptionsTransportWs/MessageType.cs

```diff
@@ -0,0 +1,92 @@
+#pragma warning disable CA1707 // Identifiers should not contain underscores
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets.SubscriptionsTransportWs;
+
+/// <summary>
+///     Protocol message types defined in
+///     https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md
+/// </summary>
+public static class MessageType
+{
+    /// <summary>
+    ///     Client sends this message after plain websocket connection to start the communication with the server
+    ///     The server will response only with GQL_CONNECTION_ACK + GQL_CONNECTION_KEEP_ALIVE(if used) or GQL_CONNECTION_ERROR
+    ///     to this message.
+    ///     payload: Object : optional parameters that the client specifies in connectionParams
+    /// </summary>
+    public const string GQL_CONNECTION_INIT = "connection_init"; // Client -> Server
+
+    /// <summary>
+    ///     The server may responses with this message to the GQL_CONNECTION_INIT from client, indicates the server accepted
+    ///     the connection.
+    /// </summary>
+    public const string GQL_CONNECTION_ACK = "connection_ack"; // Server -> Client
+
+    /// <summary>
+    ///     The server may responses with this message to the GQL_CONNECTION_INIT from client, indicates the server rejected
+    ///     the connection.
+    ///     It server also respond with this message in case of a parsing errors of the message (which does not disconnect the
+    ///     client, just ignore the message).
+    ///     payload: Object: the server side error
+    /// </summary>
+    public const string GQL_CONNECTION_ERROR = "connection_error"; // Server -> Client
+
+    /// <summary>
+    ///     Server message that should be sent right after each GQL_CONNECTION_ACK processed and then periodically to keep the
+    ///     client connection alive.
+    ///     The client starts to consider the keep alive message only upon the first received keep alive message from the
+    ///     server.
+    ///     <remarks>
+    ///         NOTE: This one here don't follow the standard due to connection optimization
```

Comment:
Is this still actual?

Reply:
I don't even understand what it says.  In any case, I'll just remove it.